### PR TITLE
added test

### PR DESCRIPTION
**Pull request by BGBroth**

### CODE REVIEW

**Review by Polyakiv-Andrey — CHANGES_REQUESTED** [review]
Tests not passed!

File: app/test_main.py [hunk 1]
```diff
@@ -1 +1,34 @@
...
+    result = can_access_google_page("https://www.google.com")
+    assert result == "Not accessible"
+
+    mock_has_internet_connection.return_value = True
+    mock_valid_google_url.return_value = False
+    result = can_access_google_page("https://www.google.com")
+    assert result == "Not accessible"
+
+    result = can_access_google_page("invalid_url")
+    assert result == "Not accessible"
```
Comment (vsmutok) [inline app/test_main.py:30]: use  `@pytest.mark.parametrize`

**Review by vsmutok — CHANGES_REQUESTED** [review]

File: app/test_main.py [hunk 2]
```diff
@@ -1 +1,30 @@
...
+        (False, True, "Not accessible", "https://www.google.com"),
+        (True, False, "Not accessible", "https://www.guagle.com"),
+        (False, False, "Not accessible", "invalid_url"),
+    ]
+)
+@patch("app.main.has_internet_connection")
+@patch("app.main.valid_google_url")
+def test_can_access_google_page(
+        mock_valid_google_url: Callable,
+        mock_has_internet_connection: Callable,
```
Comment (pavlejviki) [inline app/test_main.py:21]: wrong type annotation for mock objects

**Review by pavlejviki — CHANGES_REQUESTED** [review]

**Review by vsmutok — CHANGES_REQUESTED** [review]
please fix flake8 error

**Pull request closed without merging**